manager/state/raft/raft.go

```diff
@@ -1269,6 +1281,11 @@ func (n *Node) applyRemoveNode(cc raftpb.ConfChange) (err error) {
 	}
 
 	if cc.NodeID == n.Config.ID {
+		n.removeRaftOnce.Do(func() {
+			atomic.StoreUint32(&n.isMember, 0)
+			close(n.removeRaftCh)
+		})
```

Comment:
We should probably factor the `n.removeRaftOnce.Do` call into a function, since the same code appears in two places.